CRM/Event/Form/ParticipantView.php

```diff
@@ -180,6 +185,12 @@ public function preProcess() {
     // no contribution attached to it - maybe we have eliminated this? But I have a nasty feeling about
     // webform.
     $this->assign('totalTaxAmount', $totalTaxAmount ?? NULL);
+
+    // let's add add when tax is enabled.
```

Comment:
I think this is slightly cleaner https://github.com/civicrm/civicrm-core/pull/31540 - same thing but it just avoids calculating it incorrectly in the first place, rather than calculating it wrong & then conditionally fixing it

Reply:
fine by me :)